src/main/java/slimeknights/tconstruct/gadgets/item/slimesling/BlueSlimeSlingItem.java

```diff
@@ -0,0 +1,54 @@
+package slimeknights.tconstruct.gadgets.item.slimesling;
+
+import net.minecraft.entity.LivingEntity;
+import net.minecraft.entity.player.PlayerEntity;
+import net.minecraft.entity.player.ServerPlayerEntity;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.math.MathHelper;
+import net.minecraft.world.World;
+import slimeknights.tconstruct.common.Sounds;
+import slimeknights.tconstruct.library.SlimeBounceHandler;
+import slimeknights.tconstruct.library.network.TinkerNetwork;
+import slimeknights.tconstruct.tools.common.network.EntityMovementChangePacket;
+
+public class BlueSlimeSlingItem extends BaseSlimeSlingItem {
+
+  public BlueSlimeSlingItem(Properties props) {
+    super(props);
+  }
+
+  /** Called when the player stops using an Item (stops holding the right mouse button). */
+  @Override
+  public void onPlayerStoppedUsing(ItemStack stack, World worldIn, LivingEntity entityLiving, int timeLeft) {
+    if (!(entityLiving instanceof PlayerEntity)) {
+      return;
+    }
+
+    PlayerEntity player = (PlayerEntity) entityLiving;
+
+    // don't allow free flight when using an elytra, should use fireworks
+    if (player.isElytraFlying()) {
+      return;
+    }
+
+    float f = getForce(stack, timeLeft);
+
+    player.addExhaustion(0.2F);
+    player.getCooldownTracker().setCooldown(stack.getItem(), 3);
+    player.setSprinting(true);
+
+    float speed = f / 3F;
```

Comment:
This feels too strong. Green slime sling should be the strongest, with blue weaker but working without kicking off a block.